/**:
  um982_node:
    ros__parameters:
    
      port:
        gnss: /dev/ttyUSB0
        gnss_baudrate: 115200
        rtcm: /dev/ttyUSB1
        rtcm_baudrate: 115200
      
      heading_offset: 0.0

      heading_system: 'enu'
      
      publish:
        nmea: true

      verbose: true
